07 flex layout 2: add additional hint for cards requiring space (#118)
TheOdinProject/css-exercises: flex/07-flex-layout-2/README.md

### Hints
- You will need to change the flex-direction to push the footer down.
- You will need to add some divs as containers to get things to line up correctly.
- `flex-wrap` will help get the cards aligned correctly.

> -  Make sure cards know how much space they should take for `flex-wrap` to work as intended.

Reviewer: While I understand your reasoning for adding this hint rather than a Self Check item (and I definitely think there is merit there!), I'm not sure if this would be clear enough (not to say this is wrong; it could really be me having the trouble). What do you think of this minor tweaking of what you have:

```suggestion
-  Make sure you define how much space the cards should take up, in order for `flex-wrap` to work as intended.
```
Let me know if you feel these two statements are ultimately the same thing, or if you think one is easier to understand or anything.

Author: I love the change and I think it is a lot clearer than my original proposal. The takeaway from this wording should be "items needs their space defined for flex-wrap to work as I intend" which straight to the point.

Cards "knowing" might be confusing for plenty of people because they might not think of items as "learning" their properties from the code we write.

Thank you!
